v2/client.go

```diff
@@ -19,6 +21,8 @@ const (
 	// XBrokerAPIVersion is the header for the Open Service Broker API
 	// version.
 	XBrokerAPIVersion = "X-Broker-API-Version"
+	// XBrokerAPIOriginatingIdentity is the header for the originating identity
+	XBrokerAPIOriginatingIdentity = "X-Broker-API-Originating-Identity"
```

Comment:
I'd actually prefer to not have the constant names align so closely with the OSB API HTTP header names. By aligning it exactly with the header names people will get confused if/when OSB API changes their names - like removes the "X". By making it a bit more abstract we give ourselves the freedom to change the value of these constants w/o having to also change the name and thus break people.

e.g. just APIVersion and OriginatingIdentity should work just fine.